### ISSUE TITLE
embulk-output-redshift - no JSON support?

### ISSUE THREAD

**Issue by jdotjdot** [embulk/embulk-output-jdbc#154]
I'm using Embulk (which we just discovered today and already love it, thanks so much for your work on this!) to copy over our entire Postgres database to Redshift daily.  To do this, we've created a generic Embulk config with the table name as an environmental variable and we're just iterating over every table name in our database.

It appears that even though the Redshift output plugin is documented to support JSON, we keep getting errors that the JSON type isn't supported.  Is there something we're doing wrong?

Error:
```
...
	at usr.local.Cellar.embulk.$0_dot_8_dot_9.libexec.embulk_dot_jar.embulk.command.embulk_bundle.invokeOther34:require(file:/usr/local/Cellar/embulk/0.8.9/libexec/embulk.jar!/embulk/command/embulk_bundle.rb)
	at usr.local.Cellar.embulk.$0_dot_8_dot_9.libexec.embulk_dot_jar.embulk.command.embulk_bundle.RUBY$script(file:/usr/local/Cellar/embulk/0.8.9/libexec/embulk.jar!/embulk/command/embulk_bundle.rb:30)
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:625)
	at org.jruby.ir.Compiler$1.load(Compiler.java:111)
	at org.jruby.Ruby.runScript(Ruby.java:817)
	at org.jruby.Ruby.runScript(Ruby.java:809)
	at org.jruby.Ruby.runNormally(Ruby.java:747)
	at org.jruby.Ruby.runFromMain(Ruby.java:569)
	at org.jruby.Main.doRunFromMain(Main.java:415)
	at org.jruby.Main.internalRun(Main.java:310)
	at org.jruby.Main.run(Main.java:239)
	at org.jruby.Main.main(Main.java:201)
	at org.embulk.cli.Main.main(Main.java:23)

Error: This plugin doesn't support json type. Please try to upgrade version of the plugin using 'embulk gem update' command. If the latest version still doesn't support json type, please contact plugin developers, or change configuration of input plugin not to use json type.
```

Config:

```yaml
in:
  type: postgresql
  host: ***
  port: 5432
  user: ***
  password: ***
  database: ***
  schema: public
  ssl: true

  application_name: postgres-read-replica-input

  table: {{ env.table_name }}
  incremental: true

  column_options:
    uuid: { value_type: string }

out:
  type: redshift
  host: ***
  port: 5439
  user: ***
  password: ***
  database: dev
  schema: embulk_loaded_postgres
  table: {{ env.table_name }}

  access_key_id: ***
  secret_access_key: ***

  s3_bucket: redshift
  s3_key_prefix: embulk-loader-postgres/
  mode: replace
```

**Comment by hito4t** [comment]
This issue is same as https://github.com/embulk/embulk-output-jdbc/issues/150 .
I've created the PR https://github.com/embulk/embulk-output-jdbc/pull/152 .
I'll merge it and release new version in several days.

**Comment by jdotjdot** [comment]
Looking forward to the release; thanks!

**Comment by jdotjdot** [comment]
I'm actually encountering the same issue in `merge` mode, just FYI--I realize this PR may solve the that problem as well regardless.

**Comment by hito4t** [comment]
I've merged #150 and released!
